### ISSUE TITLE
Cargo-web support?

### ISSUE THREAD

**Issue by mvlabat** [rustwasm/wasm-pack#471]
## 💡 Feature description
Some crates may require using [cargo-web](https://github.com/koute/cargo-web) to build. For example, [stdweb](https://github.com/koute/stdweb).
Will it make sense to detect if `cargo-web` is installed and use it for compiling projects?

**Comment by ashleygwilliams** [comment]
@mvlabat - this project was primarily created to support the wasm-bindgen use case, but to the extent that it's not overly complicating i would definitely be interested in exploring supporting stdweb as well! 

if you'd like- if you could comment here with what you'd imagine the plan for integrating stdweb, we can discuss here and decide if it makes sense! i'm certainly not opposed, and i think it'd be exciting to get it to work!

**Comment by mvlabat** [comment]
@ashleygwilliams, thanks for the answer and your open position about this!

An argument for doing it (though maybe not a very strong one) is that `wasm-pack` is used in some workflows utilizing JS building tools. Don't know about Webpack ecosystem, but Parcel, for instance, has such plugin: https://github.com/catsigma/parcel-plugin-wasm.rs. It detects if `wasm-pack` is installed on user's system and if so it uses it as a default build instrument for building Rust dependencies.
So the point is that this feature could bring `cargo-web` support to JS bulding tools probably without any effort.

I think that one way of implementing this could be just to detect if `Web.toml` exists for a project. Or, if a user doesn't have any configuration, they could use some special CLI attribute that tells `wasm-pack` to use `cargo-web` binary.

Btw - about the `wasm-bindgen` use case - I've just stumbled upon this issue: https://github.com/koute/cargo-web/issues/92. It seems that exactly this use case is not currently possible. :)

**Comment by fitzgen** [comment]
I agree that https://github.com/koute/stdweb/issues/318 is the best way to unify. I think trying to wrap `cargo web` would just add more maintenance burden and should become unnecessary relatively quickly.

**Comment by ashleygwilliams** [comment]
now that https://github.com/koute/stdweb/issues/318 has landed - i *think* we can close this. are there any outstanding todos?
